plugins/node/instrumentation-runtime/src/instrumentation.ts

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InstrumentationBase } from '@opentelemetry/instrumentation';
+import { RuntimeInstrumentationConfig } from './types';
+import { VERSION } from './version';
+import {
+  EventLoopUtilization,
+  IntervalHistogram,
+  monitorEventLoopDelay,
+  performance,
+} from 'perf_hooks';
+import {
+  hrTime,
+  hrTimeDuration,
+  hrTimeToMilliseconds,
+} from '@opentelemetry/core';
+import { AttributeNames } from './enums/AttributeNames';
+import { Attributes } from '@opentelemetry/api';
+
+/**
+ * Runtime instrumentation for Opentelemetry
+ */
+export class RuntimeInstrumentation extends InstrumentationBase {
+  private eventLoopDelayHistogram: IntervalHistogram;
+  private _lastIntervalELU: EventLoopUtilization;
+  private config: RuntimeInstrumentationConfig;
+
+  constructor(
+    config: RuntimeInstrumentationConfig = {
+      monitorEventLoopDelayResolution: 10,
+    }
+  ) {
+    super('@opentelemetry/instrumentation-runtime', VERSION, config);
+    this.config = config;
+
+    // https://nodejs.org/api/perf_hooks.html#perf_hooksmonitoreventloopdelayoptions
+    this.eventLoopDelayHistogram = monitorEventLoopDelay({
+      resolution: config.monitorEventLoopDelayResolution,
+    });
+    this.eventLoopDelayHistogram.enable();
```

Comment:
Context: I am personally not a fan of the `auto-instrumentation` package and have no desire of adding this plugin there. What I am a fan of is that people hand pick the plugins that make sense to their app. 

And if they pick them, they should be active. I dont want them to pass in a "disabled" or "enabled" flag. That's why current implementation makes sense to me: You add it, you use it. And that is how it works now.

So my question is with this `enabled` / `disabled` API, how does it work? You add a plugin and somehow have to "enable" it as well, or pass in an "enabled" flag? 🤷 It makes me unsure how it would work if I made this change.

Can I suggest that if someone want to add this plugin to "auto instrumentation" that they also do any update required in this plugin? I would rather skip it unless it also makes sense for this package in isolation in this initial version.